src/components/layout/layout.js

```diff
@@ -66,6 +67,7 @@ const Layout = ({ children, displayActions, ruleUri, crumbLabel }) => {
           <Footer />
         </div>
 
+        <FloatingBubble />
         <MobileMenu isMenuOpened={isMenuOpened}></MobileMenu>
       </Auth0Provider>
     </div>
```

Comment:
Should this change be included in this PR?

Reply:
Done - this change shouldn't be here and I removed it as well